code/game/objects/items/miscellaneous.dm

```diff
@@ -180,13 +180,30 @@
 	var/itemheld = FALSE
 	var/capacity = 2
 	var/maximum_size = 2 //one human, two pets, unlimited tiny mobs, but no big boys like megafauna
+	var/kidnappingcoefficient = 1
 
+/obj/item/clothing/head/that/bluespace/attackby(obj/item/W, mob/user, params)
+	. = ..()
+	if(istype(W, /obj/item/upgradewand))
+		var/obj/item/upgradewand/wand = W
+		wand.used = TRUE
+		kidnappingcoefficient = 0.5
+		capacity = 4
+		maximum_size = 4
+		to_chat(user, "<span_class='notice'>You upgrade the [src] with the [wand]</span>")
+		playsound(user, 'sound/weapons/emitter2.ogg', 25, 1, -1)
+	
 /obj/item/clothing/head/that/bluespace/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
 	. = ..()
 	if(isliving(target))
 		var/mob/living/M = target
+		var/kidnaptime = max(10, (M.health * (M.mob_size / 2)))
+		if(iscarbon(target))
```

Comment:
Should silicons have an extra time too, if carbons get it?

Reply:
Carbon's extra time is due to how being in crit counts as negative HP

Reply:
Makes sense